tests/Aspire.Microsoft.Data.SqlClient.Tests/Aspire.Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -11,6 +11,8 @@
 
     <ProjectReference Include="..\..\src\Components\Aspire.Microsoft.Data.SqlClient\Aspire.Microsoft.Data.SqlClient.csproj" />
     <ProjectReference Include="..\Aspire.Components.Common.Tests\Aspire.Components.Common.Tests.csproj" />
+
+    <PackageReference Include="Microsoft.DotNet.RemoteExecutor" />
```

Comment:
Why are these references needed now?

Reply:
Because I cleaned the references in Aspire.Components.Common.Tests, which declared redundant/excessive dependencies.
Now Microsoft.DotNet.RemoteExecutor is only referenced where it's required, and not all projects that depend on Aspire.Components.Common.Tests.csproj require Microsoft.DotNet.RemoteExecutor.